config.yaml

```diff
@@ -0,0 +1,2 @@
+owner: elastic
```

Comment:
May you explain why this file is needed?

Reply:
This is the config file that needs to be read from root as per #87.

Reply:
Should I replace owner/repo with repository link instead in the config file? And split the information in code?

Reply:
No it's ok like this